packages/node_modules/@webex/plugin-meetings/src/stats/stats.js

```diff
@@ -0,0 +1,314 @@
+import {StatelessSparkPlugin} from '@ciscospark/spark-core';
+import uuid from 'uuid';
+
+import {MEETINGS} from '../constants';
+import StatsHistory from '../stats/history';
+import StatsAggregator from '../stats/aggregator';
+import StatsInterval from '../stats/interval';
+import StatsStream from '../stats/stream';
+import StatsFilter from '../stats/filter';
+import StatsEvents from '../stats/events';
+
+/**
+ * @class MeetingStats
+ */
+export default class MeetingStats extends StatelessSparkPlugin {
+  namespace = MEETINGS;
+
+  /**
+   * @param {Object} attrs
+   * @param {Object} options
+   * @param {Object} optionalCreateOptions {history: true/false, interval: true/false, aggregator: true/false, stream: RTCPeerConnection, filter: PeerConnection, events: true/false, id: true/false} // if filter is set do not set stream
+   */
+  constructor(attrs, options, optionalCreateOptions) {
+    super({}, options);
+    this.attrs = attrs;
+    this.options = options;
+    // usable values
+    this.history = undefined;
+    this.interval = undefined;
+    this.aggregator = undefined;
+    this.stream = undefined;
+    this.filter = undefined;
+    this.events = undefined;
```

Comment:
Same comment re: undefiend vs null. I see why you want to show this is available though

Reply:
yeah same comment as above :)